src/main/java/core/basesyntax/InfoPrinter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface InfoPrinter {
+    void getInfo();
```

Comment:
```suggestion
    void drawInfo();
```

this method is void, so we are not getting anything from it